datasets/standvastgoed/postcode_gebieden/v1.0.0.json

```diff
@@ -0,0 +1,92 @@
+{
+  "id": "postcodeGebieden",
+  "type": "table",
+  "version": "1.0.0",
+  "title": "Tabel met postcode naar gebiedsindeling",
+  "description": "Postcodes kunnen in meerdere gebieden liggen. Deze tabel is ontdubbeld. Een postcode die in meerdere buurten ligt, is toegewezen aan één buurt. Als een postcode in meerdere buurten ligt is er ontdubbeld op volgorde van: meeste hoofdadressen, meeste adressen (inclusief nevenadressen), alfabetisch op buurt code. Meer data over de gebieden is op te halen via deze api: https://api.data.amsterdam.nl/v1/docs/datasets/gebieden.html",
+  "schema": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "additionalProperties": false,
+    "identifier": "postcode",
+    "required": [
+      "schema",
+      "postcode"
+    ],
+    "display": "postcode",
+    "properties": {
+      "schema": {
+        "$ref": "https://schemas.data.amsterdam.nl/schema@v1.1.1#/definitions/schema"
+      },
+      "peildatum": {
+        "type": "string",
+        "format": "date",
+        "title": "Peildatum"
+      },
+      "postcode": {
+        "type": "string",
+        "title": "Postcode"
+      },
+      "gebiedBuurtCode": {
+        "type": "string",
```

Comment:
Het is evt. mogelijk om hier (en ook voor `wijk` en `ggwGebied`) een relatie van te maken naar de `gebieden` dataset.

Reply:
Zie: https://schemas.data.amsterdam.nl/docs/ams-schema-spec.html#relatering

Reply:
relaties toegevoegd https://github.com/Amsterdam/amsterdam-schema/pull/705/commits/c1134a4678cab22f45bd58a13f866547af1810cb